src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -0,0 +1,195 @@
+<h3>Bestandsliste</h3>
+
+@code {
+
+// Parameter Items for asset list
+    [Parameter] public string[] Items { get; set; }
+
+    // TODO: remove the code for the initialization
+    private string[] _examples =
+    {
+        "Trockeneis", "Ethanol", "Natriumhydroxid", "Aceton", "Ammoniumchlorid", "Wasserstoffperoxid",
+        "Kaliumpermanganat", "Messzylinder", "Erlenmeyerkolben", "Reagenzglas"
+    };
+
+    protected override void OnInitialized()
+    {
+        Items = _examples;
+    }
+
+}
+
+
+@* grid with one column for items *@
+<div class="container text-center">
+    <div class="col">
+        @foreach (var item in Items)
+        {
+            <div
+                class="row custom-checkbox-container">
+                <p>@item</p>
+                <label class="custom-checkbox">
+                    <input type="checkbox"
+                           class="form-check-input"
+                           aria-label="...">
+                    <span class="checkmark"> </span>
+                </label>
+            </div>
+        }
+    </div>
+</div>
+
+@* pagination *@
+<nav
+    aria-label="Page navigation example">
+    <ul
+        class="pagination">
+        <li
+            class="page-item disabled">
+            <a class="page-link">
+                🡸
+            </a>
```

Comment:
🌱 We can think about transforming the `a` tags into buttons to use `@onclick` later for actual page navigation

Reply:
You can do this, but we can also do it when it's needed